### PR TITLE
Add settings sections

### PR DESCRIPTION
**Pull request by huwshimi**
Done:
- Add the nav items and base components for each section of the settings. Fixes: https://github.com/canonical-web-and-design/maas-squad/issues/1332.

QA:
- Load the app.
- You should see the nav items seen here: https://app.zeplin.io/project/5ce5423a25863c1e0d7abb27/screen/5d1cbd18160fa1aff58546d0
- Clicking on them should show the title of the section.

### CODE REVIEW

File: ui/src/app/settings/components/Nav/Nav.js [hunk 1]
```diff
@@ -37,10 +37,55 @@ const _generateSection = (section, location) => {
 
 export const Nav = props => {
   const nav = [
+    {
+      url: "/configuration",
```
Comment (steverydz) [inline ui/src/app/settings/components/Nav/Nav.js]: would `path` be a better name for this?

File: ui/src/app/settings/components/Nav/__snapshots__/Nav.test.js.snap [hunk 2]
```diff
@@ -7,6 +7,69 @@ exports[`Nav renders 1`] = `
   <ul
     className="settings-nav__list"
   >
+    <li
+      className="settings-nav__item"
+      key="/configuration"
+    >
+      <strong
+        className=""
```
Comment (steverydz) [inline ui/src/app/settings/components/Nav/__snapshots__/Nav.test.js.snap]: empty `className`

File: ui/src/app/settings/components/Nav/__snapshots__/Nav.test.js.snap [hunk 3]
```diff
@@ -21,33 +84,204 @@ exports[`Nav renders 1`] = `
...
+          Storage
+        </Link>
+      </strong>
+    </li>
+    <li
+      className="settings-nav__item"
+      key="/network"
+    >
+      <strong
+        className=""
```
Comment (steverydz) [inline ui/src/app/settings/components/Nav/__snapshots__/Nav.test.js.snap]: empty `className`

File: ui/src/app/settings/components/Nav/__snapshots__/Nav.test.js.snap [hunk 4]
```diff
@@ -21,33 +84,204 @@ exports[`Nav renders 1`] = `
...
+          </HashLink>
+        </li>
+      </ul>
+    </li>
+    <li
+      className="settings-nav__item"
+      key="/scripts"
+    >
+      <strong
+        className=""
```
Comment (steverydz) [inline ui/src/app/settings/components/Nav/__snapshots__/Nav.test.js.snap]: empty `className`

File: ui/src/app/settings/components/Nav/Nav.js (hunk 1 above)
Comment (huwshimi) [reply ui/src/app/settings/components/Nav/Nav.js]: Good idea, thanks.

File: ui/src/app/settings/components/Nav/__snapshots__/Nav.test.js.snap (hunk 2 above)
Comment (huwshimi) [reply ui/src/app/settings/components/Nav/__snapshots__/Nav.test.js.snap]: I think these are OK, as they will only get a value when the item is active.

**Pull request merged**